plugins/webhooks/index.js

```diff
@@ -634,53 +631,64 @@ async function createEvents(eventFactory, userFactory, pipelineFactory,
     const events = [];
     const meta = createMeta(parsed);
 
-    for (let i = 0; i < pipelines.length; i += 1) {
-        const p = pipelines[i];
-        /* eslint-disable no-await-in-loop */
-        const pipelineBranch = await p.branch;
```

Comment:
It is good to follow eslint, but `pipeline.branch` seems not to use asynchronous processing? I think it just wraps synchronouse processing with Promise(my understanding may be something wrong).
https://github.com/screwdriver-cd/models/blob/master/lib/pipeline.js#L925

So, I think fixing those doesn't improve performance than we think.
[IMO] And I like small changes if it's needless.

Reply:
Never mind, I found there are some await statement except of `p.branch` in this loop.